recipes/mygeotab/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mygeotab" %}
+{% set version = "0.5.4" %}
+{% set sha256 = "48e29238da3ce8c236475fcf144fafeda785b3d8c7660beb9ded88732e0e63dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
It looks like there is an `entry_point` defined in the `setup.py`:

https://github.com/Geotab/mygeotab-python/blob/master/setup.py#L47-L49

Please add the entry point to the `build` section as described here:
https://conda.io/docs/building/meta-yaml.html#python-entry-points

and add a test in the testing section as shown here:
https://conda.io/docs/building/meta-yaml.html#test-commands